pyomo/common/unittest.py

```diff
@@ -500,6 +500,11 @@ def shortDescription(self):
         # Disable nose's use of test docstrings for the test description.
         return None
 
+    def currentTestPassed(self):
+        # Note: this only works for Python 3.4+
```

Comment:
Do we need this comment considering we're only supporting 3.6+ now?

Reply:
It is not strictly necessary, but I have taken to leaving breadcrumbs when I have to dig around for a solution, especially if web searches turn up a number of alternative "solutions" (and the first 3 are all for Python<3.3).